### DISCUSSION TITLE
Where can i find grafana dashboard json for milvus standalone version 2.4?

### DISCUSSION THREAD

**Discussion by king1302217** [milvus-io/milvus#32962] [Q&A and General discussion]
https://github.com/milvus-io/milvus/blob/master/deployments/monitor/grafana/milvus-dashboard.json
This config is for k8s version. But my version is milvus standalone(2.4).
So where can i find grafana dashboard json for milvus standalone version 2.4?

**Comment by yhmo** [comment]
https://github.com/milvus-io/milvus/blob/2.4/deployments/monitor/grafana/milvus-dashboard.json

Currently, the milvus-dashboard.json is no different for 2.3 and 2.4.

**Reply by king1302217** [reply]
@yhmo But it is suit for  k8s version, not standalone version. Where can i find the dashboard.json for standalond version?

**Reply by yhmo** [reply]
The monitoring service only supports Milvus Cluster. We don't have monitoring solution for Milvus Standalone.

**Reply by xiaofan-luan** [reply]
https://github.com/milvus-io/milvus/issues/32981#issuecomment-2105614535

I think @zhuwenxing mentioned, it is already supported

**Reply by yhmo** [reply]
The Milvus online doc should describe how to use it for standalone.

**Comment by xiaofan-luan** [comment]
maybe there should be a easy modify on that
